### PR TITLE
Add background animation to page

### PR DESCRIPTION
**Pull request by iceicery**
This pull request is for discussing an unsolved bug.

### CODE REVIEW

File: src/components/Layout/Layout.css [hunk 1]
```diff
@@ -9,6 +9,7 @@
   margin: 0 auto;
   position: relative;
   padding: 0 16px;
+  z-index: -2;
```
Comment (julieg18) [inline src/components/Layout/Layout.css:12]: The bug is caused by add a `z-index: -2` to the entire page.

File: src/components/BgAnimation/BgAnimation.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ export default function BgAnimation() {
       >
         <g opacity="0.15">
           <path
-            fill-rule="evenodd"
-            clip-rule="evenodd"
+            fillRule="evenodd"
```
Comment (iceicery) [inline src/components/BgAnimation/BgAnimation.js:10]: Nice fix. Sorry that I always forget to change the name to React format.

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -16,8 +16,20 @@
   color: rgba(255, 255, 255, 0.5);
 }
 
+.header__is-main .title_size_l {
+  z-index: 1;
```
Comment (iceicery) [inline src/components/Header/Header.css:23]: I guess we need to set the form z-index:2
image

**Review by iceicery — COMMENTED** [review]
The animation improved a lot. The page looks beautiful. Set z-index:2 for the form and I think the website is ready to go!!!!

File: src/components/Header/Header.css (hunk 3 above)
Comment (julieg18) [reply src/components/Header/Header.css:23]: Nice job spotting that!

**Review by julieg18 — APPROVED** [review]
Great job!

**Pull request merged**